src/Config/Converters/EntityRestOptionsConverter.cs

```diff
@@ -108,13 +108,21 @@ public override void Write(Utf8JsonWriter writer, EntityRestOptions value, JsonS
             writer.WriteNull("path");
         }
 
-        writer.WriteStartArray("methods");
-        foreach (SupportedHttpVerb method in value.Methods)
+        if (value.Methods is not null)
         {
-            writer.WriteStringValue(JsonSerializer.SerializeToElement(method, options).GetString());
+            writer.WriteStartArray("methods");
+            foreach (SupportedHttpVerb method in value.Methods)
+            {
+                writer.WriteStringValue(JsonSerializer.SerializeToElement(method, options).GetString());
+            }
+
+            writer.WriteEndArray();
+        }
+        else if (value.Methods is null && options.DefaultIgnoreCondition != JsonIgnoreCondition.WhenWritingNull)
+        {
+            writer.WriteNull("methods");
         }
```

Comment:
Removed the condition.

We do set `DefaultIgnoreCondition` as `JsonIgnoreCondition.WhenWritingNull` currently. Even if we change it to something else in the future, it would be better to not write anything to the config file than writing `null` values.